network/peer/peer.go

```diff
@@ -635,21 +646,18 @@ func (p *peer) sendNetworkMessages() {
 			knownPeersFilter, knownPeersSalt := p.Config.Network.KnownPeers()
 			msg, err := p.Config.MessageCreator.GetPeerList(knownPeersFilter, knownPeersSalt)
 			if err != nil {
-				p.Log.Error("failed to create get peer list message",
+				p.Log.Error(failedToCreateMessageLog,
 					zap.Stringer("nodeID", p.id),
+					zap.Stringer("messageOp", message.GetPeerListOp),
 					zap.Error(err),
 				)
-				continue
+				return
```

Comment:
In most places we closed the connection after failing to build a message... This PR standardizes that behavior.